code/addons/test/src/manager-store.ts

```diff
@@ -1,8 +1,19 @@
-import { experimental_UniversalStore } from 'storybook/internal/manager-api';
+import {
+  experimental_UniversalStore,
+  experimental_getStatusStore,
+} from 'storybook/internal/manager-api';
 
-import { type StoreState, storeOptions } from './constants';
+import {
+  STATUS_TYPE_ID_A11Y,
+  STATUS_TYPE_ID_COMPONENT_TEST,
+  type StoreState,
+  storeOptions,
+} from './constants';
 
 export const store = experimental_UniversalStore.create<StoreState>({
   ...storeOptions,
   leader: (globalThis as any).CONFIG_TYPE === 'PRODUCTION',
 });
+
+export const componentTestStatusStore = experimental_getStatusStore(STATUS_TYPE_ID_COMPONENT_TEST);
+export const a11yStatusStore = experimental_getStatusStore(STATUS_TYPE_ID_A11Y);
```

Comment:
Because `addon-test` both handles CT statuses and a11y statuses, we get a status store slice for each of the two type ids, that operates independently of each other.